### ISSUE TITLE
Tip vorticies using ALM compared to a resolved wing 

### ISSUE THREAD

**Issue by samfredriksson** [turbinesFoam/turbinesFoam#256]
I have been doing some comparisons with a single wing (ALM and resolved) in a steady state plug flow using simpleFoam and kOmegaSST. I find that the forces (sources) is calculated as expected but the results with ALM have much less vorticity and especially the tip vortices are much stronger in the resolved case than in the case using ALM. This is perhaps not so surprising but I have not find any good reference cases covering tip vortices using ALM, resolved cases and experiments. Is this a know problem with the model and are there any ways to increase the tip vorticity and is there some literature covering this?

**Comment by petebachant** [comment]
Turning off any end effects model would probably increase the tip vortex strength. Beyond that, I'm not sure. Googling "tip vortex actuator line", it looks like there are some people who have looked at it though.

**Comment by andersgoude** [comment]
This can be related to the smoothing parameter epsilon being too large, as it spreads the vorticity over a larger area. How large is your epsilon, compared to the chord of the blade?

**Comment by samfredriksson** [comment]
The chord is between 3.3 m and 0.53 m. I have tested different epsilon, both the default and with fixed epsilon for the complete profile and then from 0.625 to 0.078125 and in-between.

**Comment by andersgoude** [comment]
Did lower epsilon give any improvements?

One should know that the ALM will have issues modelling wing tips, as the no-penetration boundary condition is not satisfied, which is why we need the tip corrections. Still, to some part, the flow solver can handle the 3D effects by itself, which causes the tip corrections to partly do what the flow solver is doing, and you compensate twice for the same thing in a way

**Comment by samfredriksson** [comment]
Yes, there certainly is a tip vortex but it is seen to be much smaller than for the resolved case. By lowering epsilon, the maximum abs(Qmax-Qmin) as well as the magVorticity increase substantially. It is, however, mainly locally around the actuator line and does not affect the tip vorticity strength significantly. In both cases the actuator line is discretized by nElements 160 with a span width of 12 m.

**Comment by andersgoude** [comment]
You have quite a low aspect ratio, which will be a tricky case for the ALM. However, am I correct in assuming that it is only the vorticity of the tip vortex that is incorrect, but also the circulation that is lower for the ALM than in the refined case? It sounds strange if the forces are correct, given the close coupling between the forces and the circulation. If the circulation is significantly different, it is something one should investigate more in detail.

**Comment by samfredriksson** [comment]
Thanks. It seems that the output is more as expected running pimpleFoam compared to simpleFoam although the pimpleFoam case is run semistationary, that is everything including the inlet boundary condition as constant. Do you have any comments on this? Otherwise I will just continue to work with pimpleFoam for the time being.

**Comment by petebachant** [comment]
That's interesting. I suppose there is a time dependent nature of the vortices in reality, but the mean flows would be the same.

**Comment by samfredriksson** [comment]
I agree that it should be the same, which also was the reason I ran it in steady state.
